docs/source/40_deploying_ADAM.md

```diff
@@ -1,57 +1,106 @@
+Prior to following these instructions, you need to already have setup your AWS account and know your
+AWS access keys.  See https://aws.amazon.com/ for details.
+
+#### Configure CGCloud
+
+Begin by reading the CGcloud [readme](https://github.com/BD2KGenomics/cgcloud).
+
+Next, configure [CGCloud core](https://github.com/BD2KGenomics/cgcloud/blob/master/core/README.rst) and then install the [CGcloud spark plugin](https://github.com/BD2KGenomics/cgcloud/blob/master/spark/README.rst).
+
+Note, the steps to register your ssh key and create the template boxes below need only be done once.
+```
+cgcloud register-key ~/.ssh/id_rsa.pub
+cgcloud create generic-ubuntu-trusty-box
+cgcloud create -IT spark-box
+```
+
+#### Launch a cluster
+
+Spin up a Spark cluster with one master and two slave nodes with the command:
+```
+cgcloud create-cluster spark -s 2 -t m3.large
+```
+
+Once running, you can ssh to `spark-master` with the command:
+```
+cgcloud ssh spark-master
+```
+
+
+Spark is already installed on the `spark-master` machine and slaves, test it by starting a spark-shell
+```
+spark-shell
+exit()
 ```
 
-If you want to use spot pricing, add `--spot-price` as an option to
-`spark_ec2_launch` (below) and `export MY_SPOT_PRICE=1.399`.
+#### Install ADAM
 
-Export the path to your `spark-ec2` script,
+To use the ADAM application on top of Spark, we need to download and install ADAM on `spark-master`
+From the command line on `spark-master` download a release from:
+https://github.com/bigdatagenomics/adam/releases
 
-```bash
-export SPARK_EC2_SCRIPT="/path/to/spark-0.8.1/ec2/spark-ec2"   # CHANGE ME
+As of this writing, CGCloud supports Spark 1.6.2, not Spark 2.x, so download
+the Spark 1.x Scala2.10 release:
+```
+wget https://repo1.maven.org/maven2/org/bdgenomics/adam/adam-distribution_2.10/0.20.0/adam-distribution_2.10-0.20.0-bin.tar.gz
+tar -xvfz adam-distribution_2.10-0.20.0-bin.tar.gz
 ```
 
-Set up some aliases for commands to the spark ec2 script,
+You can now run `./bin/adam-submit` and `./bin/adam-shell` using your EC2 cluster.
+
+#### Input and Output data on HDFS and S3
+
+Spark requires a file system, such a HDFS or a network file mount, that all machines can access.
+The CGCloud EC2 Spark cluster you just created is already running HDFS.
 
-```bash
-alias spark_ec2_launch="$SPARK_EC2_SCRIPT -k $MY_KEYPAIR \
--i $MY_KEYFILE -s $MY_CLUSTER_SIZE --zone us-east-1c \
---instance-type=$MY_INSTANCE_TYPE launch $MY_CLUSTER_NAME"
-alias spark_ec2_stop="$SPARK_EC2_SCRIPT stop $MY_CLUSTER_NAME"
-alias spark_ec2_start="$SPARK_EC2_SCRIPT -i $MY_KEYFILE start $MY_CLUSTER_NAME"
-alias spark_ec2_destroy="$SPARK_EC2_SCRIPT destroy $MY_CLUSTER_NAME"
-alias spark_ec2_login="$SPARK_EC2_SCRIPT -k $MY_KEYPAIR -i $MY_KEYFILE login $MY_CLUSTER_NAME"
+The typical flow of data to and from your ADAM application on EC2 will be:
+- Upload data to AWS S3
+- Use Conductor (described below) or otherwise transfer from S3 to the HDFS on your cluster
```

Comment:
Can you add an anchor link `{#conductor}` in the section where conductor is described, and link from here `(described below)` -> `[(described below)](#conductor)`. This'll make navigation a bit easier.